### PR TITLE
changes

### PR DESCRIPTION
**Pull request by SMOLKEYS**

### CODE REVIEW

File: mod-src/build.gradle.kts [hunk 1]
```diff
@@ -20,6 +20,9 @@ plugins {
 /** The output jar files will contain this string in their names. */
 val jarName = "compiled-mod"
 
+/** Windows sucks. */
+val windows = System.getProperty("os.name").lowercase().contains("windows")
```
Comment (Mnemotechnician) [inline mod-src/build.gradle.kts]: Use `System.getProperty("os.name").contains("windows", true)` or `"windows" in System.getProperty("os.name").lowercase()`, please

File: mod-src/build.gradle.kts [hunk 2]
```diff
@@ -130,16 +133,19 @@ task("jarAndroid") {
 		println("${reusable.size} dependencies are already desugared and can be reused.")
 		if (needReDex.isNotEmpty()) println("Desugaring ${needReDex.size} dependencies.")
 
-		// for every non-reusable dependency, invoke d8 and save the new hash
+		// for every non-reusable dependency, invoke d8 (d8.bat for windows) and save the new hash
+
+		val d8 = if(windows) "d8.bat" else "d8"
```
Comment (Mnemotechnician) [inline mod-src/build.gradle.kts]: there should be a space after if (not trying to pick at your commit, it just hurts me to see this)

**Review by Mnemotechnician — CHANGES_REQUESTED** [review]

File: mod-src/build.gradle.kts (hunk 2 above)
Comment (SMOLKEYS) [reply mod-src/build.gradle.kts]: `if (`?

Comment (Mnemotechnician) [reply mod-src/build.gradle.kts]: yeah...

File: mod-src/build.gradle.kts (hunk 1 above)
Comment (SMOLKEYS) [reply mod-src/build.gradle.kts]: quiche

**Pull request merged**